test/pretty-print.test.js

```diff
@@ -1,9 +1,9 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const boot = require('..')
 
-test('pretty print', t => {
+test('pretty print', async t => {
```

Comment:
```suggestion
test('pretty print', (t, done) => {
```

Can we keep using the callback api and call `done` inside?

Reply:
> Can we keep using the callback api and call done inside?

yes, we can also use callback api, I did a commit here: [4fc9916](https://github.com/fastify/avvio/pull/312/commits/4fc99166babb0c52412da840e209dc57dc5d37cf)


the first commit is with Promises the second is with callbacks, so there are two ways, to do the test 😉